<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkBooleanOperationsWidgetControls</class>
 <widget class="QWidget" name="QmitkBooleanOperationsWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QmitkDataSelectionWidget" name="dataSelectionWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="differenceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Difference</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/BooleanOperationsWidget.qrc">
         <normaloff>:/MultiLabelSegmentationUtilities/BooleanOperations/BooleanDifference_48x48.png</normaloff>:/MultiLabelSegmentationUtilities/BooleanOperations/BooleanDifference_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="intersectionButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Intersection</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/BooleanOperationsWidget.qrc">
         <normaloff>:/MultiLabelSegmentationUtilities/BooleanOperations/BooleanIntersection_48x48.png</normaloff>:/MultiLabelSegmentationUtilities/BooleanOperations/BooleanIntersection_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="unionButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Union</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/BooleanOperationsWidget.qrc">
         <normaloff>:/MultiLabelSegmentationUtilities/BooleanOperations/BooleanUnion_48x48.png</normaloff>:/MultiLabelSegmentationUtilities/BooleanOperations/BooleanUnion_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>273</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataSelectionWidget</class>
   <extends>QWidget</extends>
   <header>internal/Common/QmitkDataSelectionWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/BooleanOperationsWidget.qrc"/>
 </resources>
 <connections/>
</ui>
